python/tests/lang/test_language_context.py

```diff
@@ -3,6 +3,7 @@
 from aac.lang.active_context_lifecycle_manager import get_initialized_language_context
 from aac.lang.language_context import LanguageContext
 from aac.spec import get_aac_spec, get_primitives, get_root_keys
+from aac.plugins.validators.required_fields import get_required_fields
```

Comment:
Related to my other comment about moving the location of `get_required_fields`; The tests exercising the method `get_required_fields` should be in the corresponding test file. If you move the method out of the validator plugin, you can ignore this comment.

Reply:
We talked about this. Going to resolve this comment.